admin/client/App/screens/List/index.js

```diff
@@ -333,25 +333,25 @@ const ListView = React.createClass({
 			checkedItems: {},
 		});
 	},
-	deleteTableItem (item, e) {
-		if (e.altKey) {
+	deleteTableItem (item, event) {
+		if (event.altKey) {
 			this.props.dispatch(deleteItem(item.id));
 			return;
 		}
 
-		e.preventDefault();
+		event.preventDefault();
 
 		this.setState({
 			confirmationDialog: {
 				isOpen: true,
 				label: 'Delete',
 				body: (
-					<p>
+					`<p>
 						Are you sure you want to delete <strong>${item.name}</strong>?
```

Comment:
The back ticks were added in the hopes that Babel would then leap into action and complete the previous partial solution. Reversion is in place and I've retested simply removing the errant legacy `$`, and that change is ready to be committed.